build/README.md

```diff
@@ -7,13 +7,13 @@ that compiles to JavaScript) in the right order.
 
 The polyPod core code, as well as all the bundled features, are separated into
 several NPM packages that include each other via file URLs. Unfortunately, NPM
-does, at the time of writing, not support building a tree of dependent packages,
+does, at the time of writing, not support building a tree of dependent packages, so
 it's up to us to build everything in the right order.
 
 We experimented with a few tools that should eliminate all, or at least some of
 the logic in this module, such as Yarn and pnpm workspaces, and while we could
 get those to work with a few workarounds, in the end we decided to stick with
-plain NPM. It's probably not the best package manager, but the most widely
+plain `npm`. It's probably not the best package manager, but the most widely
```

Comment:
I don't care too much, but I think NPM - as in referring to the tool, not a particular CLI client for it, was actually more clear.